app/src/main/res/layout/fragment_event_detail.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
ConstraintLayout in ConstraintLayout isn't recommended situation. Is there any better way?

Reply:
If ConstrainsLayout is used somewhere, we shouldn't need another layout type. That's because you can position any view wherever you want. ConstraintLayout provides this functionality with its powerful attributes.